api/database/migrations/2025_02_14_073642_add_partial_submissions_to_form_submissions.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('form_submissions', function (Blueprint $table) {
+            $table->enum('status', ['partial', 'completed'])
```

Comment:
Let's add const to FormSubmission for these 2 values, and use the const instead of the string

Reply:
It's there already! Let's use it